datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/manifest.json

```diff
@@ -1,35 +1,44 @@
 {{
-  "display_name": "{integration_name}",
-  "maintainer": "{email}",
-  "manifest_version": "1.0.0",
-  "name": "{check_name}",
-  "metric_prefix": "{check_name}.",
-  "metric_to_check": "",
-  "creates_events": false,
-  "short_description": "",
-  "guid": "{guid}",
-  "support": "{support_type}",
-  "supported_os": [
-    "linux",
-    "mac_os",
-    "windows"
-  ],
-  "public_title": "{integration_name}",
-  "categories": [
-    ""
-  ],
-  "type": "check",
-  "is_public": false,
-  "integration_id": "{integration_id}",{pricing_plan}{terms}{author_info}
+  "manifest_version": "2.0.0",
+  "app_uuid": "{guid}",
```

Comment:
Can we change the name of the variable from `guid` to `app_uuid`?
I think this variable should solely be used for this field, and if there comes a time when another guid is needed for something, it should likely be unique